app/Jobs/ImportTurboVotePosts.php

```diff
@@ -94,33 +116,11 @@ public function handle(Rogue $rogue)
                     ]);
 
                     if (! $post['data']) {
-                        $tvCreatedAtMonth = strtolower(Carbon::parse($record['created-at'])->format('F-Y'));
-                        $sourceDetails = isset($referralCodeValues['source_details']) ? $referralCodeValues['source_details'] : null;
-                        $postDetails = $this->extractDetails($record);
-
-                        $postData = [
-                            'campaign_id' => (int) $referralCodeValues['campaign_id'],
-                            'campaign_run_id' => (int) $referralCodeValues['campaign_run_id'],
-                            'northstar_id' => $user->id,
-                            'type' => 'voter-reg',
-                            'action' => $tvCreatedAtMonth . '-turbovote',
-                            'status' => $this->translateStatus($record['voter-registration-status'], $record['voter-registration-method']),
-                            'source' => 'turbovote',
-                            'source_details' => $sourceDetails,
-                            'details' => $postDetails,
-                        ];
-
-                        try {
-                            $post = $rogue->createPost($postData);
-
-                            if ($post['data']) {
-                                $this->stats['countPostCreated']++;
-                            }
-                        } catch (\Exception $e) {
-                            info('There was an error storing the post for: ' . $record['id'], [
-                                'Error' => $e->getMessage(),
-                            ]);
-                        }
+                        CreateTurboVotePostInRogue::dispatch($record, $referralCodeValues, $user);
+
```

Comment:
@katiecrane I'm trying to make sure the `$this->stats['countPostCreated']+++` stat increases when a new post is made. I tried to do the `Queue::after` method above (lines 67-76_ but that didn't work/not sure I'm using it correctly. Do you have any ideas on how we can still get this stat count to work?

Reply:
@chloealee how are you running your queue locally?

Reply:
OHH good call, @katiecrane ! I was doing `sync` but changed to `redis` and ran `php artisan queue:listen` but it timedout :-/ and the file I'm using is a test one with only 18 records 🤔

Reply:
@chloealee ah I thought an issue could have been using `php artisan queue:work` which I believe only runs what's in the queue when you use the command and doesn't grab subsequent jobs...I ran into an issue with that when working on the queue stuff for longshot

Reply:
but I've also not had luck with using `queue::after`

Reply:
ah interesting! I tried with `php artisan queue:work` too but the script got `Killed`. I'll keep trying other ways - basically just need to figure out how to tell the first job that the job to create the post in Rogue was successful or not !

Reply:
@chloealee hmmm interestinggggg I wonder if the second job can edit the stat itself?

Reply:
@katiecrane yes! I originally tried to pass the stat to the job and have it increase but it didn't work the way I tried to do it. Then I thought about it more and realized that it probably actually makes sense for that counter to stay in the original job since it is one data point in the overall stat?

Reply:
@chloealee were you calling `queue::after` from a service provider?

Reply:
@katiecrane I'm not! I saw that but tried to do from this job (lines above) instead. Maybe that is why it isn't working tho! If I call it from the [service provider](https://github.com/DoSomething/chompy/blob/master/app/Providers/AppServiceProvider.php) though, is that strange for it to increase the post created counter from there?

Reply:
@chloealee I'm not sure if it could update the stat from there, unless it somehow knew which stat it was?

Reply:
@katiecrane hm yeah that is what I'm thinking too... I'll try to dig around for something besides `queue::after` bc I'm not sure if this route will work!